dht/dhtcore/NodeStore.c

```diff
@@ -1001,6 +1001,8 @@ static struct Node_Link* discoverLinkC(struct NodeStore_pvt* store,
     }
 
     struct Node_Two* parent = closest->child;
+    uint64_t realPathParentChild = pathParentChild == 1 ?
+        closest->cannonicalLabel : pathParentChild;
```

Comment:
Is it possible for pathParentChild to be 1 while ```parent != child``` ?
If so then I think this is another bug.